app/utils.ts

```diff
@@ -663,6 +663,8 @@ export const currencyToCountryCode = (currency: string) => {
   const currencyOverrides: Record<string, string> = {
     XAF: "cm", // Central African CFA Franc (Cameroon)
     XOF: "sn", // West African CFA Franc (Senegal)
+    UGX: "ug", // Uganda
+    TZS: "tz", // Tanzania
```

Comment:
you can remove these as they follow the regular format, we don't need to override.